iib/web/models.py

```diff
@@ -837,7 +839,12 @@ def from_json(cls, kwargs, batch=None):
 
         cls._from_json(
             request_kwargs,
-            additional_optional_params=['from_index', 'organization', 'bundles'],
+            additional_optional_params=[
+                'from_index',
+                'organization',
+                'bundles',
+                'omps_operator_version',
```

Comment:
Why are we allowing the user to specify `omps_operator_version` when we don't use it while calling OMPS? Am I missing something?